frontend/src/components/listeners/SubmitExtraction.tsx

```diff
@@ -39,21 +42,32 @@ const widgets = {
 
 export default function SubmitExtraction(props: SubmitExtractionProps) {
 
-
 	const {fileId, datasetId, open, handleClose, selectedExtractor} = props;
 	const dispatch = useDispatch();
 
 	const submitFileExtraction =
 		(fileId: string | undefined, extractorName: string | undefined, requestBody: FormData) => dispatch(submitFileExtractionAction(fileId, extractorName, requestBody));
 	const submitDatasetExtraction =
 		(datasetId: string | undefined, extractorName: string | undefined, requestBody: FormData) => dispatch(submitDatasetExtractionAction(datasetId, extractorName, requestBody));
-	const onSubmit = (formData: FormData) => {
+   
+	const jobSummaryFetch = (job_id: string | undefined) => dispatch(fetchJobSummary(job_id));
+	const jobUpdatesFetch = (job_id: string | undefined) => dispatch(fetchJobUpdates(job_id));
+    
```

Comment:
Maybe you can move those to the extractorStatus component.  This way you don't need to pass them in as props